hist/hist/src/TH1.cxx

```diff
@@ -5486,7 +5486,7 @@ Long64_t TH1::Merge(TCollection *li)
 Bool_t TH1::Multiply(TF1 *f1, Double_t c1)
 {
    if (!f1) {
-      Error("Add","Attempt to multiply by a non-existing function");
```

Comment:
This should be changed to Multiply. Right?

Reply:
Indeed :-)